src/incipit/processing.py

```diff
@@ -128,6 +128,35 @@ def split_regions_by_image(
     return regions_by_image_index
 
 
+def adjust_region_position_by_image(
+        regions: typing.List[typing.Union[Region, IndexedRegion]],
+        expand_vertical_ratio_of_system: float = 0.5,
+        expand_vertical_ratio_of_gap: float = 0.45,
+)-> typing.List[typing.Union[Region, IndexedRegion]]:
```

Comment:
@Allan518 This needs to have a docstring that describes, even roughly, how to use it, or else it will be forgotten.

And as I say this, I notice you have a poor role model since none of my existing code has docstrings! I guess I was always postponing it to later — or rather I was expecting to refactor the package seriously soon. But we should probably go around and add documentation.

In this docstring, you need to make clear how you are using the leftmost and rightmost boundaries. I think I understand what you are doing (taking a host of systems, and making sure they start and end at the same place roughly, to detect anomalies of width), but maybe describing it would make clearer what heuristics is, and how it is tunable.

Reply:
@jlumbroso I have added docstring for the function, please review.